userapi/storage/postgres/deltas/2025011001110000_add_xsigning_updatable_without_uia_before_ms.go

```diff
@@ -0,0 +1,23 @@
+package deltas
```

Comment:
As mentioned OOB, we probably don't need this migration and can instead use https://github.com/element-hq/dendrite/blob/f4506a0d8236d58bd72c835bbd6e6adc972fca1f/clientapi/routing/register.go#L57-L69 instead? (same goes for the SQLite migration etc.)

Reply:
Done. Added a new map to the structure for managing this kind of session. Also deleted `xsigning_updatable_without_uia_before_ms` column from the database and reverted all related logic